### PR TITLE
fix long lines and Documentation

### PR DESCRIPTION
**Pull request by jaybatra26**
Signed off by Jay Batra <jaybatra73@gmail.com>

Pull request title
------------------

The format of the Pull request title must be:

    <component>: <short_change_message> #<issue number>

If you add a [github-recognised keyword](https://help.github.com/articles/closing-issues-using-keywords/) then
the associated issue can be closed automatically once the pull request is merged, e.g.::

    <component>: <short_change_message> Fix #<issue number>

Valid component names are listed in the `label list <https://github.com/rucio/rucio/labels>`_.

### CODE REVIEW

File: doc/source/installing_server.rst [hunk 1]
```diff
@@ -11,7 +11,9 @@ Install via pip
 
 Heads up: We recommend to use the docker-based install (see next section) as it will configure many things for you automatically. Only use the pip-based install if you have a good reason and know how to configure your webservices manually:
 
-``pip install rucio``
+.. code-block:: php
```
Comment (mlassnig) [inline doc/source/installing_server.rst:14]: i'm not entirely sure why you chose "php" instead of "bash" :-)

**Review by mlassnig — COMMENTED** [review]
did you use the `tools/submit-pull-request` script to submit this pull request? i see the template message in the PR description

**Comment by jaybatra26** [comment]
Not really, I will remove the template message. Also, I just noticed that my sister's git is configured in the terminal. I didn't realize that all my commits are going by her name. I will close this PR and re-raise a new one.

**Comment by mlassnig** [comment]
closed due to wrong client configuration

**Pull request closed without merging**